content/blog/2021/01/2021-01-26-new-ebook-build-deploy-and-automate-great-stuff-with-jenkins.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "New eBook: Build, deploy and automate great stuff with Jenkins"
+tags: 
+- jenkins
+- jenkinsistheway
+- casestudies
+author: alyssat
+---
+
+In April of last year, we launched a new Jenkins community website called https://jenkinsistheway.io/[JenkinsIsTheWay], named after the phrase oft-repeated in the Disney TV program, The Mandalorian. The Mandalorian, a Star Wars adventure, follows the travails of a lone gunfighter in the galaxy's outer reaches, far from the authority of the New Republic.
```

Comment:
Do we want it to be explicit? Trademark concerns still apply there IMHO

Reply:
Simplified and removed the referenced text in 0adbd977466394d8fbece2a3702b6b16950d6d42.  Thanks to @alyssat for pairing with me during the edits.